__tests__/app.test.js

```diff
@@ -121,4 +121,72 @@ describe("200: /api/articles", () => {
         });
       });
   });
+  describe("POST: /api/articles/:article_id/comments", ()=>{
+    test('should return an array of 1 element', () => {
+      return request(app)
+      .post("/api/articles/5/comments")
+      .send({
+        username: "rogersop",
+        body: "Hello I am new Here"
+      })
+      .expect(201)
+      .then(({body})=>{
+        const {comment} = body;
+        expect(comment.length).toBe(1)
+      })
+    })
+    test('should return the posted comment', () => {
+      return request(app)
+      .post("/api/articles/5/comments")
+      .send({
+        username: "rogersop",
+        body: "Hello I am new Here"
+      })
+      .expect(201)
+      .then(({body})=>{
+        const {comment} = body;
+        expect(comment[0].body).toBe("Hello I am new Here")
```

Comment:
As you are posting a single comment here it might be better to return just the comment object on that key rarther than an array containing one item.